packages/react-slider/src/components/slider.stories.tsx

```diff
@@ -29,7 +31,60 @@ const meta: Meta<typeof RecommendationsSlider> = {
 export default meta;
 type Story = StoryObj<typeof RecommendationsSlider>;
 
-export const Default: Story = {};
+function isRecoVisible(reco: HTMLElement): boolean {
+  const track = reco.parentElement;
+  const recoRect = reco.getBoundingClientRect();
+  const trackRect = track.getBoundingClientRect();
+
+  return recoRect.right > trackRect.left && recoRect.left < trackRect.right;
+}
+
+export const Default: Story = {
+  play: async ({ canvasElement, step }) => {
+    const sliderEl = canvasElement
+      .querySelector('tfup-recommendations-slider').shadowRoot;
+
+    await waitFor(function recosLoaded() {
+      expect(sliderEl.querySelector('tfup-recommendation')).toBeTruthy();
+    }, {timeout: 2000});
+
+    const [ arrowRight, arrowLeft ] = [
+      sliderEl.querySelector<HTMLButtonElement>('[aria-label="arrow-right"]'),
+      sliderEl.querySelector<HTMLButtonElement>('[aria-label="arrow-left"]'),
+    ];
+    const [ firstReco, secondReco, thirdReco ] = Array
+      .from(sliderEl.querySelectorAll('tfup-recommendation'));
```

Comment:
This should not be needed for destructuring. NodeList implements iteration and any iterable can be deconstructed.
```suggestion
    const [
      firstReco,
      secondReco,
      thirdReco,
    ] = sliderEl.querySelectorAll('tfup-recommendation');
```

Reply:
> Type 'NodeListOf<Recommendation>' is not an array type.ts(2461)

Reply:
This must be because of the compilation target. I guess its ok